src/message/message.controller.ts

```diff
@@ -75,8 +75,8 @@ export class MessageController {
     const table = restaurant.tables.find((t) => t._id.equals(tableId));
 
     const text = dto.order.reduce((previousValues, currentValue) => {
-      return previousValues + `${currentValue.name}(${currentValue.count}), `;
-    }, `person: ${dto.personCount}, ${table.name} - `);
+      return previousValues + `\n${currentValue.name} ${currentValue.count}`;
+    }, '\n' + `person: ${dto.personCount}, ${table.name}`);
```

Comment:
why U need concat in this case?

Reply:
@roobel2005  Change '\n' + `person: ${dto.personCount}, ${table.name}`);
To  `\n person: ${dto.personCount}, ${table.name}`);